src/app/games/pylos/tests/PylosMove.spec.ts

```diff
@@ -68,9 +68,8 @@ describe('PylosMove', () => {
         expect(move.equals(otherMove3)).toBeFalse();
         expect(move.equals(otherMove4)).toBeFalse();
     });
-    it('Should consider captures [a, b] and [b, a] equal', () => {
+    it('Should encode [low, high] as [high, low]', () => {
```

Comment:
attention qu'`encode` a une signification très particulière, plutôt "should represent"?

Reply:
fait!